src/python/pants/option/subsystem.py

```diff
@@ -205,10 +216,13 @@ def _construct_subsystem_rule(cls) -> Rule:
             parameters=FrozenDict(),
             awaitables=(
                 AwaitableConstraints(
-                    rule_id=None,
+                    rule_id="pants.engine.internals.options_parsing.scope_options",
```

Comment:
We switched this call to be by-name at runtime, so we have to update this "rule graph construction time" info to match.